### ISSUE TITLE
Quadz is incompatible with latest version of gecko.

### ISSUE THREAD

**Issue by chris3000** [ethanejohnsons/FPV-Racing-Mod-Fabric#26]
I installed the latest version of gecko with the latest released version of Quadz, and got stuck in an endless loop where the Mojang loading screen would keep reappearing. When I looked at the logs, I saw an error with quadz trying to access a class in geckolib3. I kept downgrading gecko until Quadz successfully installed. The version that worked was geckolib 3.0.42, which is also labeled [DO NOT USE].

**Comment by chris3000** [comment]
[14:44:43] [main/INFO]: Caught error loading resourcepacks, removing all selected resourcepacks
java.util.concurrent.CompletionException: java.lang.IncompatibleClassChangeError: Expected static method software.bernie.geckolib3.geo.render.GeoBuilder.constructGeoModel(Lsoftware/bernie/geckolib3/geo/raw/tree/RawGeometryTree;)Lsoftware/bernie/geckolib3/geo/render/built/GeoModel;
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:708) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:690) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:703) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_4014.method_18365(class_4014.java:71) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3414/672319620.run(Unknown Source) ~[?:?]
	at net.minecraft.class_1255.execute(class_1255.java:94) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014.method_18367(class_4014.java:70) ~[client-intermediary.jar:?]
	at net.minecraft.class_4014$$Lambda$3181/287790979.execute(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(CompletableFuture.java:529) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:653) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:474) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:561) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:635) ~[?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442) ~[?:1.8.0_51]
	at net.minecraft.class_1060$$Lambda$3417/1273876465.execute(Unknown Source) ~[?:?]
	at com.mojang.blaze3d.systems.RenderSystem.replayQueue(RenderSystem.java:116) ~[client-intermediary.jar:?]
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:103) ~[client-intermediary.jar:?]
	at net.minecraft.class_1041.method_15998(class_1041.java:308) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1068) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:681) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:215) [client-intermediary.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.IncompatibleClassChangeError: Expected static method software.bernie.geckolib3.geo.render.GeoBuilder.constructGeoModel(Lsoftware/bernie/geckolib3/geo/raw/tree/RawGeometryTree;)Lsoftware/bernie/geckolib3/geo/render/built/GeoModel;
	at dev.lazurite.quadz.common.data.util.TemplateResourceLoader.loadGeo(TemplateResourceLoader.java:74) ~[quadz-1.0.1.jar:?]
	at dev.lazurite.quadz.common.data.util.TemplateResourceLoader.load(TemplateResourceLoader.java:55) ~[quadz-1.0.1.jar:?]
	at dev.lazurite.quadz.common.data.util.TemplateResourceLoader$$Lambda$3467/247402957.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_51]
	at dev.lazurite.quadz.common.data.util.TemplateResourceLoader.method_14491(TemplateResourceLoader.java:45) ~[quadz-1.0.1.jar:?]
	at net.minecraft.class_4013.method_29490(class_4013.java:15) ~[client-intermediary.jar:?]
	at net.minecraft.class_4013$$Lambda$3184/2037162244.run(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:705) ~[?:1.8.0_51]
	... 244 more

**Comment by The-PPWD** [comment]
The latest version of Quadz that was released was for 1.16.x (we're slow :P). Current development efforts are focused towards 1.18.x, so we're not likely to do backwards patches. We compiled the last release against GeckoLib 3.0.34 which I believe was the newest at the time. I don't really want to say "we aren't willing to do that" given that this is a perfectly valid issue, but given that we're already late to update, we'd rather just focus on the updating efforts given that the newest GeckoLib at the time of release worked. Sorry.
